src/components/Calculator.js

```diff
@@ -1,62 +1,58 @@
-          <button className="btn-opr" type="button" onClick={hendleClick}><span>5</span></button>
-          <button className="btn-opr" type="button" onClick={hendleClick}><span>6</span></button>
-          <button className="btn-opr-right" type="button" onClick={hendleClick}><span>-</span></button>
-        </div>
-        <div className="button">
-          <button className="btn-opr" type="button" onClick={hendleClick}><span>1</span></button>
-          <button className="btn-opr" type="button" onClick={hendleClick}><span>2</span></button>
-          <button className="btn-opr" type="button" onClick={hendleClick}><span>3</span></button>
-          <button className="btn-opr-right" type="button" onClick={hendleClick}><span>+</span></button>
-        </div>
-        <div className="button">
-          <button className="btn-opr-zero" type="button" onClick={hendleClick}><span>0</span></button>
-          <button className="btn-opr" type="button" onClick={hendleClick}><span>.</span></button>
-          <button className="btn-opr-right" type="button" onClick={hendleClick}><span>=</span></button>
-        </div>
+const Calculator = () => {
+  const [state, setState] = useState({
+    total: 0,
+    next: null,
+    operation: null,
+  });
+  const hendleClick = (e) => {
+    setState({ ...state, ...calculate(state, e.target.innerHTML) });
+  };
 
+  return (
+    <div className="calculator-cont">
+      <div className="results">
+        <p className="result">
+          {state.total}
+          {state.operation}
+          {state.next}
+        </p>
+      </div>
+      <div className="button">
+        <button className="btn-opr" type="button" onClick={hendleClick}><span>AC</span></button>
+        <button className="btn-opr" type="button" onClick={hendleClick}><span>+/-</span></button>
+        <button className="btn-opr" type="button" onClick={hendleClick}><span>%</span></button>
+        <button className="btn-opr-right" type="button" onClick={hendleClick}><span>÷</span></button>
+      </div>
+      <div className="button">
+        <button className="btn-opr" type="button" onClick={hendleClick}><span>7</span></button>
+        <button className="btn-opr" type="button" onClick={hendleClick}><span>8</span></button>
+        <button className="btn-opr" type="button" onClick={hendleClick}><span>9</span></button>
+        <button className="btn-opr-right" type="button" onClick={hendleClick}><span>x</span></button>
+      </div>
+      <div className="button">
+        <button className="btn-opr" type="button" onClick={hendleClick}><span>4</span></button>
+        <button className="btn-opr" type="button" onClick={hendleClick}><span>5</span></button>
+        <button className="btn-opr" type="button" onClick={hendleClick}><span>6</span></button>
+        <button className="btn-opr-right" type="button" onClick={hendleClick}><span>-</span></button>
       </div>
```

Comment:
- [ ] Your buttons seem to have a CSS style of `30vh` which means each button takes 30% of the screen height. That causes your calculator to exceed the max screen size which does not conform the wirerame provided. Kindly fix this issue so your styles resemble the wireframe provided.

![image](https://user-images.githubusercontent.com/90556221/182831587-6cdf0854-5ee7-4248-9fad-57e52b46d4bf.png)